### PR TITLE
Remove assert and log the elapsed time for the test

### PR DESCRIPTION
**Pull request by saurabh500**
Removing assert and logging the time taken to run the tests so that the logs can be inspected to see what's going on in the CI with the actual timeout.

### CODE REVIEW

**Comment by saurabh500** [comment]
To be used to fix/debug https://github.com/dotnet/corefx/issues/26382

File: src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs [hunk 1]
```diff
@@ -107,7 +107,8 @@ public void ConnectionTimeoutTestWithThread()
             }
 
             int threshold = (timeoutSec + 1) * 1000;
-            Assert.True(theMax < threshold);
+
+            Console.WriteLine($"Elapsed Time in ConnectionTimeoutTestWithThread  {theMax}");
```
Comment (corivera) [inline src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs]: Can you print out the threshold also?

**Comment by JonHanna** [comment]
Test NETFX x86 Release Build
"MSBUILD : error MSB4166: Child node "3" exited prematurely. Shutting down. Diagnostic information may be found in files in the temporary files directory named MSBuild_*.failure.txt." https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/7099/console

**Comment by Priya91** [comment]
@danmosemsft The CI passed, is this good to merge?

**Comment by Priya91** [comment]
Merging to unblock PRs.

**Pull request merged**